README.md

```diff
@@ -314,6 +288,137 @@ folders. To test if everything works open the demo.html file.
 If everything works, add a README.md file to the component folder and add
 a documentation page to the docs/ folder.
 
+You can use the [coherent-guic-cli](https://github.com/CoherentLabs/GameUIComponents/tree/master/tools/cli#getting-started) to build single component.

```

Comment:
Can we move this in the beginning of the explanation on how to add a component? I think it will be more beneficial to start how to do it with the cli, then how to do it manually.

Reply:
Yes. Moved.